### PR TITLE
new icon: hadoop (original, plain)

### LINKED ISSUE #584: Icon request: Apache Hadoop
### Links
https://hadoop.apache.org/

### PR DESCRIPTION
**Pull request by MAGistR-bit**
new icon: hadoop (original, plain)

**This PR closes #584**

### CODE REVIEW

**Comment by MAGistR-bit** [comment]
Hi! How long to wait for verification?

**Comment by Snailedlt** [comment]
> Hi! How long to wait for verification?

Depends, usually it takes under a week before it's reviewed by at least one maintainer and just over a week before it's merged into develop

File: package.json
Comment (Snailedlt) [inline package.json:3]: Please don't update the package-lock and package files :)

File: package-lock.json
Comment (Snailedlt) [inline package-lock.json:3]: Please don't update the package-lock and package files :)

Looks like it might have happened because you based your branch on master instead of develop. Try checking out develop instead, and cherry-picking this commit: https://github.com/devicons/devicon/pull/1438/commits/a28d5ad0f21ceae5ed56ac6f697f3141ae5cc465

File: devicon.json
Comment (Snailedlt) [inline devicon.json:2146]:
```suggestion
       "altnames": ["Apache Hadoop"],
```

File: devicon.json
Comment (Snailedlt) [inline devicon.json:2163]: Hmm, do you think maybe the line icon can also be considered a plain icon? 
If so, you can change the filename to `hadoop-plain.svg` and add this:
```suggestion
  	"aliases": [
            {
                "base": "plain",
                "alias": "line"
            }
        ] 
```

File: icons/hadoop/hadoop-line.svg
Comment (Snailedlt) [inline icons/hadoop/hadoop-line.svg:5]: plain and line icons should have only one path if possible. So please check if the paths can be merged into one :)

File: icons/hadoop/hadoop-original-wordmark.svg
Comment (Snailedlt) [inline icons/hadoop/hadoop-original-wordmark.svg:4]: Ehh, this is just a png image inside a SVG, so it's not a real vector image. Please convert into SVG, you can see how to convert PNG's to SVGs here: https://github.com/devicons/devicon/pull/1428#issuecomment-1271702666

File: icons/hadoop/hadoop-original.svg
Comment (Snailedlt) [inline icons/hadoop/hadoop-original.svg:1]: This one looks good to me, but it should be minimized. I recommend you check out our [Recommended resources and tools](https://github.com/devicons/devicon/wiki/Recommended-resources-and-tools). Specifically you should use SVGOMG for this, and follow the following guide: https://github.com/devicons/devicon/issues/1279#issuecomment-1195433256

**Review by Snailedlt — CHANGES_REQUESTED** [review]
A few things to sort out here, but the icons look good for the most part. I especially liked the line icon :)

**Comment by Snailedlt** [comment]
@MAGistR-bit Let me know if you need any help :)

**Comment by MAGistR-bit** [comment]
> @MAGistR-bit Let me know if you need any help :)
Hello, [@Snailedlt! ](https://github.com/Snailedlt)
Made adjustments to the work:
1. I left two versions: original, plain.
2. I optimized the SVG code using SVGOMG.
3. Updated devicon.json. 

P. S. I had errors when I submitted work to a remote repository, so I probably made unnecessary commits.

File: devicon.json
Comment (Snailedlt) [inline devicon.json:3325]: A few things need to be fixed:
- Indentation
- Remove `original` from `font`
- Change main color to the correct color: `#ffff00` (found in the official logo here: https://www.apache.org/logos/#hadoop)

(Note that the fill color also needs to be changed from `#feeb50` to `#ffff00` in [hadoop-original.svg](https://github.com/devicons/devicon/pull/1438/files#diff-6b3aedfa818bbe99cb6ac25a2d782b005498a9fe5ac775a517568641fe0942b6)

If you want, you can just apply the suggestion here (and change the [hadoop-original.svg](https://github.com/devicons/devicon/pull/1438/files#diff-6b3aedfa818bbe99cb6ac25a2d782b005498a9fe5ac775a517568641fe0942b6) file) 👇 
```suggestion
       "tags": [
      "framework",
      "big data"
      ],
      "versions": {
        "svg": [ 
          "original",
          "plain"
        ],
        "font": [ 
          "plain"
        ]
      },
      "color": "#ffff00", 
      "aliases": [
        {
          "base": "plain",
          "alias": "line"
        }	
      ] 
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Nice, you fixed a lot in those commits, well done! Don't worry about the extra commits, since it'll all be squashed when it's merged into develop.

Some changes are still needed in `devicon.json`, just apply the suggestions and it should be good.


## hadoop-wordmark
I also found the official wordmark logo here: https://www.apache.org/logos/#hadoop
So we can add that to the PR too :)

Logo link 👇 
https://svn.apache.org/repos/asf/comdev/project-logos/originals/hadoop.svg

Logo Preview 👇

**Comment by MAGistR-bit** [comment]
@Snailedlt I have edited the color of the image (hadoop-original.svg).
In addition, I have applied your suggestions. **Jørgen**, did I do everything right? 

You found the official wordmark logo, cool! 👍
Can we add this logo later?

File: devicon.json
Comment (Snailedlt) [inline devicon.json:3325]: Indentation

```suggestion
    {
        "name": "hadoop",
        "altnames": [
            "Apache Hadoop"
        ],
        "tags": [
            "framework", 
            "big data"
        ],
        "versions": {
            "svg": [
                "original", 
                "plain"
            ],
            "font": [
                "plain"
            ]
        },
        "color": "#ffff00",
        "aliases": [
            {
                "base": "plain",
                "alias": "line"
            }
        ]
    },
```

**Comment by Snailedlt** [comment]
@MAGistR-bit I just reformated the devicon.json... looks good now. However I think we need to make a new PR, since the check-bot failed due to wrong PR title (there was an extra space), and it keeps using the old one even after I changed it.

So, please make a new PR. I'll be closing this one, and waiting for a new one to open. Thanks in advance 🙇

**Pull request closed without merging**
